docs/setup.rst

```diff
@@ -79,6 +79,44 @@ Docker Compose
 Now proceed to `"Setup" <#setup>`__.
 
 
+Heroku
+------
```

Comment:
I don't understand the issue here. Is there a migration path that is handled in other deployments but not by this?
I haven't read anything about migrations in the docs, is there somewhere I can look at to understand the issue?